view/frontend/templates/js/replacejs.phtml

```diff
@@ -753,85 +755,56 @@ ob_start();
                         var BC = BoltCheckout.configure(cart, hintBarrier.promise, callbacks);
                 });
             } else {
```

Comment:
in separate PR could we refactor this if/else - maybe we can have paymentOnly logic in one function and multistep logic in another

Reply:
Completely agree, created task for this
https://boltpay.atlassian.net/browse/MX-1277
CC: @usmanismail-bolt